Add an env reader that checks for secret files (#424)
sloria/environs: README.md

# 'TTL': 42}
```

## Reading Docker-style secret files

Some values should not be stored in the environment. For this use case a commonly

> used technique is to store the value (f.i. a password) in a file, and set the path

Reviewer: what's "f.i."?

```suggestion
used technique is to store the value (e.g., a password) in a file and set the path
```

> to that file in an environment variable. Use `FileAwareEnv` in place of `Env` and
> it will automatically check for environment variables with `_FILE` appended. If the

Reviewer: ```suggestion
to that file in an environment variable. Use `FileAwareEnv` in place of `Env` to automatically check for environment variables with the `_FILE` suffix. If the
```
